python/lvmdrp/core/sky.py

```diff
@@ -260,22 +262,32 @@ def skymodel_pars_from_header(header, telescope):
     # define astros
     s, m = obs.observe(sun).apparent(), obs.observe(moon).apparent()
 
-    # define target
-    target_ra, target_dec = ra * u.deg, dec * u.deg
-    target = Star(ra_hours=target_ra.to(u.hourangle), dec_degrees=target_dec.to(u.deg))
-    t = obs.observe(target).apparent()
+    # define targets for sci, skye, skyw
+    sci_target_ra, sci_target_dec = sci_ra * u.deg, sci_dec * u.deg
+    sci_target = Star(ra_hours=sci_target_ra.to(u.hourangle), dec_degrees=sci_target_dec.to(u.deg))
+    sci_t = obs.observe(sci_target).apparent()
+    skye_target_ra, skye_target_dec = skye_ra * u.deg, skye_dec * u.deg
+    skye_target = Star(ra_hours=skye_target_ra.to(u.hourangle), dec_degrees=skye_target_dec.to(u.deg))
+    skye_t = obs.observe(skye_target).apparent()
+    skyw_target_ra, skyw_target_dec = skyw_ra * u.deg, skyw_dec * u.deg
+    skyw_target = Star(ra_hours=skyw_target_ra.to(u.hourangle), dec_degrees=skyw_target_dec.to(u.deg))
+    skyw_t = obs.observe(skyw_target).apparent()
```

Comment:
can you add a space or a comment between each telescope chunk, to break this up and make it more readable?

Reply:
Done!  Also ended up adding some try/except here, in case the ra/dec are Null